src/merge.py

```
@@ -83,68 +83,68 @@ def validate_2fa(api: PyiCloudService) -> bool:
         security_key_names = api.security_key_names
 
         if security_key_names:
-            print_step(TAG_2F_AUTH, f"Security key confirmation is required. Please plug in one of the following keys: {', '.join(security_key_names)}", one_liner=True)
+            print_step(TAG_2F_AUTH, f"Security key confirmation is required. Please plug in one of the following keys: {', '.join(security_key_names)}", True)
```

Ruff (0.15.9)

[warning] Boolean positional value in function call

(FBT003)